docs/en/stack/ml/df-analytics/ml-dfa-regression.asciidoc

```diff
@@ -24,7 +24,12 @@ the field you want to predict.
 //tag::regression-algorithms[]
 {regression-cap} uses an ensemble learning technique called extreme gradient 
 boost (XGboost) which combines decision trees with gradient boosting 
```

Comment:
Here again, we use a technique **similar to** the extreme gradient boost**ing**.